meteor/lib/lib.ts

```diff
@@ -18,10 +18,8 @@ export * from '@sofie-automation/corelib/dist/lib'
 
 /**
  * Convenience method to convert a Meteor.call() into a Promise
- * @param  {string} Method name
- * @return {Promise<any>}
  */
-export async function MeteorPromiseCall(callName: string, ...args: any[]): Promise<any> {
+export async function MeteorPromiseCall<T>(callName: string, ...args: any[]): Promise<T> {
```

Comment:
In this scenario no. The intention was to use the type argument when calling `MeteorPromiseCall` to decrease the number of places where we work with `any` typed values. But since it hasn't been done, it can be reverted.

Reply:
With a background in C# i can say that getting back a type of eg. User, instead of object, makes it much cleares what is being worked on, and allows intellisense to aid in further development.
I am not that familier with Sofie yet to say how many places would need updating to accomedate this change, but for readability and continued development, id say it would be a great help to know the type being worked on.

Reply:
In general I would agree with you that stronger typings is better, we do have a low priority task to fix up some of the usages of any too.

This particular case stood out as odd to me, as I know that we are calling it in 2 places. One of those is a unit test, the other is some js 'magic' which is using so much `any` internally, that this wont benefit it in any way.
I'm mostly just curious as to whether this was to solve anything or if it was simply spotted whilst in the area.

Reply:
Following the discussion above, I'll revert this since the generic `<T>` does not actually improve the types.